src/Uno.Foundation/Rect.cs

```diff
@@ -99,6 +99,10 @@ public Rect(Point point1, Point point2)
 
 		public bool IsEmpty => Empty.Equals(this);
 
+		internal bool IsInfinite => Infinite.Equals(this);
+
+		internal bool IsUniform => Left == Top && Left == Right && Left == Bottom;
```

Comment:
I think you should use an epsilon here.

Reply:
In this case it is ported from the C++ code - I can add the epsilon there, it is not present in the original source though